docs/HowToUseRamdump.md

```diff
@@ -54,10 +54,16 @@ copying ramdump_0x02020000_0x0210c800.bin to  $TIZENRT_BASEDIR/build/output/bin
 
 ## Ramdump Parsing Steps
 1. Run Ramdump Parser Script
+1-1. CUI
 ```
 cd $TIZENRT_BASEDIR/tools/ramdump/
 python dumpParce.py -r $TIZENRT_BASEDIR/build/output/bin/ramdump_0x02020000_0x0210c800.bin -e $TIZENRT_BASEDIR/build/output/bin/tinyara -g 0
 ```
+1-2. GUI
```

Comment:
Please write more details.
1. What is different between them.
2. User can execute one among them, not all of them.